aws_quota/check/iam.py

```diff
@@ -96,22 +110,120 @@ class AttachedPolicyPerUserCheck(InstanceQuotaCheck):
     service_code = 'iam'
     quota_code = 'L-4019AD8B'
     used_services = [service_code]
+    concurrency = 32
 
     @staticmethod
     def get_all_identifiers(session: boto3.Session) -> typing.List[str]:
-        return [user['UserName'] for user in session.client('iam').list_users()['Users']]
+        if session.region_name == 'us-east-1':
```

Comment:
The IAM is a global service. It can be accessed from any region and in any region the results are exactly the same. However, only in us-east-1 the Service Quota returns the limit value for IAM. In all other regions the quota value is missing for IAM. It seems that us-east-1 is the "main" region in some sense.

I'll add comment, thanks!